srcpkgs/rustup/template

```diff
@@ -12,7 +13,7 @@ maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="Apache-2.0, MIT"
 homepage="https://www.rustup.rs"
 distfiles="https://github.com/rust-lang/${pkgname}/archive/${version}.tar.gz"
-checksum=ad46cc624f318a9493aa62fc9612a450564fe20ba93c689e0ad856bff3c64c5b
+checksum=0203231bfe405ee1c7d5e7e1c013b9b8a2dc87fbd8474e77f500331d8c26365f
 
 post_build() {
 	# rustup errors out because it doesn't know about armv6l-musl
```

Comment:
If it won't be built for armv6l-musl anymre, the special casing around it below should be fixed as well